tests/v0_tests/test_index.py

```diff
@@ -189,4 +192,14 @@ def run():
             return True
         assert run()
 
-
+    def test_create_custom_number_of_replicas(self):
+        intended_replicas = 5
+        settings = {
+            "number_of_replicas": intended_replicas
+        }
+        self.client.create_index(index_name=self.index_name_1, settings_dict = settings)
+        resp = requests.get(
```

Comment:
could you just get this from the Marqo [get index settings endpoint](https://docs.marqo.ai/0.0.15/API-Reference/settings/)? We don't want any interaction with OpenSearch in py-marqo/ py-marqo tests:

Reply:
updated